stock_storage_type/models/stock_location.py

```diff
@@ -4,9 +4,8 @@
 
 from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
-from odoo.osv.expression import AND, OR
```

Comment:
IMO, let's try to keep these operators when building complex domain for the sake of understandability :wink: